index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Verifiable Credentials API v0.3</title>
+  <title>Verifiable Credentials API v0.4</title>
```

Comment:
```suggestion
  <title>Verifiable Credentials API v0.5</title>
```

We should probably jump to 0.5, because some people think that even numbers mean stable releases.

Reply:
Does this work for you, @dlongley ?